app/javascript/.storybook/preview.js

```diff
@@ -117,4 +44,26 @@ export const parameters = {
   html: {
     root: '#story-content',
   },
```

Comment:
(blocking) - this needs to be removed now - it was added before to make sure the generated HTML code didn't include the theme switcher. Now that we've also removed the `div` with `id="story-content"` the HTML preview is broken, but if we just remove this whole chunk it'll all work fine 👍 
```suggestion
```